.github/workflows/e2e-full.yml

```diff
@@ -25,7 +25,9 @@ jobs:
           node-version: 20
 
       - name: Install
-        run: npm ci
+        run: |
+          npm ci 
+          npx playwright install --with-deps chromium
```

Comment:
installing chromium only when

Reply:
see https://playwright.dev/docs/browsers#google-chrome--microsoft-edge
we can run against the Chrome that is surely available on CI .. let's try it at least
certainly should not be installing _every_ time we run a job (surely?)